app/src/pages/_app.tsx

```diff
@@ -62,19 +63,20 @@ function App({ Component, pageProps }: AppProps) {
           /** Consistent navbar shows available resources as regular content page changes to drill into details */
           navbar={
             <Navbar width={{ base: '320px' }}>
-              {router.pathname.startsWith('/authoring') ? (
+              {router.pathname.startsWith('/authoring') && (
                 <>
                   <Navbar.Section pt={18}>
                     <Center>
-                      <Text c="gray">Draft Artifacts</Text>
+                      <Text c="gray">Browse Measure Repository</Text>
```

Comment:
```suggestion
                      <Text c="gray">Draft Artifacts</Text>
```
I think we want to keep this what it was before so that's it's clear they are draft artifacts and not artifacts in the measure repository.